### DISCUSSION TITLE
Per-repository configuration

### DISCUSSION THREAD

**Discussion by patrick-5546** [ibhagwan/fzf-lua#1590] [Q&A] [answered]
I am working in a monorepo and get this message: 
```
git status' took 2.47 seconds, consider using `git_icons=false` in this repository or use `silent=true` to supress this message.
```

I was able to disable it globally with
```lua
opts { files { git_icons = false }}
```

But the message seems to imply that it is possible to disable git icons just for a particular repository. How can that be done?

**Comment by ibhagwan** [comment, marked as answer]
Change your opts to a lua function, check the cwd as condition, rough sketch:
```lua
lua require('fzf-lua').files(function()
  local cwd = vim.uv.cwd()
  if cwd == `foo`
    return {}
  else
    return { git_icons = true }
  end
end)
```

**Reply by ibhagwan** [reply]
Btw, there are quite  a few cross platform utilities for working with paths/git, etc check out the code in path.lua and reuse whichever you want the module is exported as `require('fzf-lua').path`.
